astropy/modeling/tests/test_functional_models.py

```diff
@@ -153,6 +153,16 @@ def test_Shift_inverse():
     assert_allclose(m.inverse(m(6.789)), 6.789)
 
 
+def test_Scale_fittable():
+    m = models.Scale(1.2)
+    assert(m.fittable==True)
+
+
+def test_Shift_fittable():
+    m = models.Shift(1.2)
+    assert(m.fittable==True)
+
+
```

Comment:
Fittable is one thing but are they also fitted *correctly*? It is nice to have a test for that. So further down the line, if fitting for a compound model failed, we can say that "oh surely it can't be Shift because we have a unit test for that which passed".